Documentation/cluster-crd.md

```diff
@@ -72,7 +72,7 @@ Below are the settings available, both at the cluster and individual node level,
 
 ### Placement Configuration Settings
 
-Placement configuration for the cluster services. It includes the following keys: `api`, `mds`, `mon`, `osd`, `rgw` and `all`. Each service will have its placement configuration generated by merging the generic configuration under `all` with the most specific one (which will override any attributes).
+Placement configuration for the cluster services. It includes the following keys: `api`, `mds`, `mgr`, `mon`, `osd`, `rgw` and `all`. Each service will have its placement configuration generated by merging the generic configuration under `all` with the most specific one (which will override any attributes).
```

Comment:
Great catch. Actually, the `mds` and `rgw` placements are part of the file and object CRDs, respectively. Could you remove those two from this list as well? Thanks!